README.md

```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
     On Debian/Ubuntu:
     
     ```bash
-    $ sudo apt install libpq-dev
+     sudo apt install libpq-dev
```

Comment:
Well, it is a commit practice to use `$` to indicate that the command should be run on behalf of a regular user.

Reply:
Oh ok, but this is a pain when copy/paste.
Sorry for this commit.